sklearn/linear_model/tests/test_least_angle.py

```diff
@@ -76,13 +77,39 @@ def test_simple_precomputed():
             assert ocur == X.shape[1]
 
 
+def _assert_same_lars_path_result(output1, output2):
+    assert_equal(len(output1), len(output2))
+    for o1, o2 in zip(output1, output2):
+        assert_array_equal(o1, o2)
```

Comment:
Make this assert_allclose just in case there is some discrepancy or loss of numerical precision on some platforms